src/chains/ethereum_1/ethChainState.ts

```diff
@@ -153,85 +153,9 @@ export class EthereumChainState {
-
-    if (waitForConfirm !== ConfirmType.None && waitForConfirm !== ConfirmType.After001) {
-      throwNewError(`Specified ConfirmType ${waitForConfirm} not supported`)
-    }
-
-    const { headBlockNumber: preCommitHeadBlockNum } = await this.getChainInfo()
-
     return new Promise((resolve, reject) => {
-      let getBlockAttempt = 1
-      // get the chain's current head block number...
-      const { processed, transaction_id: transactionId } = transactionResponse || {}
-      // starting block number should be the block number in the transaction receipt. If block number not in transaction, use preCommitHeadBlockNum
-      const { block_num: blockNum = preCommitHeadBlockNum } = processed || {}
-      const startingBlockNumToCheck = blockNum - 1
-
-      let blockNumToCheck = startingBlockNumToCheck
-      let inProgress = false
-
-      // Keep reading blocks from the chain (every checkInterval) until we find the transationId in a block
-      // ... or until we reach a max number of blocks or block read attempts
-      const timer = setInterval(async () => {
-        try {
-          if (inProgress) return
-          inProgress = true
-          const hasReachedConfirmLevel = await this.hasReachedConfirmLevel(
-            blockNumToCheck,
-            transactionId,
-            waitForConfirm,
-          )
-          if (hasReachedConfirmLevel) {
-            this.resolveAwaitTransaction(resolve, timer, transactionResponse)
-          }
-          blockNumToCheck += 1
-        } catch (error) {
-          const mappedError = mapChainError(error)
-          if (mappedError.name === 'BlockDoesNotExist') {
-            // Try to read the specific block - up to getBlockAttempts times
-            if (getBlockAttempt >= maxBlockReadAttempts) {
-              this.rejectAwaitTransaction(
-                reject,
-                timer,
-                'maxBlockReadAttemptsTimeout',
-                `Await Transaction Failure: Failure to find a block, after ${getBlockAttempt} attempts to check block ${blockNumToCheck}.`,
-              )
-              return
-            }
-            getBlockAttempt += 1
-          } else {
-            // re-throw error - not one we can handle here
-            throw mappedError
-          }
-        } finally {
-          inProgress = false
-        }
-        if (blockNumToCheck > startingBlockNumToCheck + blocksToCheck) {
-          this.rejectAwaitTransaction(
-            reject,
-            timer,
-            'maxBlocksTimeout',
-            `Await Transaction Timeout: Waited for ${blocksToCheck} blocks ~(${(checkInterval / 1000) *
-              blocksToCheck} seconds) starting with block num: ${startingBlockNumToCheck}. This does not mean the transaction failed just that the transaction wasn't found in a block before timeout`,
-          )
-        }
-      }, checkInterval)
+      // Will be implemented
```

Comment:
When something needs to be coded, it should have a TODO prefix in the comment so we can find it
If features are not supported or implemented, the code should throw the appropriate message/error so developers know that the code isn't working as he might expect.